voting.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Voting
+sidebar_label: Voting
+slug: /More on Governance
+---
+
+Anyone who holds UMA tokens is invited to vote on any improvement proposals or liquidations through the UMA Voter Dapp. 
+
+Once the wallet holding UMA is connected to the Dapp, the voting process consists of three stages: the commit phase, where votes and made; the reveal phase where votes are revealed and tallied, and finally voters who have both committed and revealed their vote may claim their UMA voting rewards if their vote was congruent with the majority.
+
+For UMIPs, the vote is a binary Yes/No question on whether the improvement proposal should be accepted; for disputes it is necessary to calculate the value under dispute of the price identifier at the relevant time and enter that figure into the voting Dapp.
+
+Both the commit phase and the reveal phase last 24 hours and voters must both commit and reveal in order to have their votes counted and be considered for the voting rewards.  Once the vote is finalised, these rewards can be claimed from the voting Dapp.

```

Comment:
Let's link to this tutorial as well

https://docs.umaproject.org/uma-tokenholders/voter-dApp

Reply:
Link added